content/index.html

```diff
@@ -768,16 +913,15 @@ <h2>Evidence Quality – Superior, Strong, and Fair</h2>
         </p>
       </section>
       <section>
-        <h2>Use of DEA Signing Certificate as Superior Evidence</h2>
+        <h2>Examples of the Use of One Piece of Strong Evidence</h2>
```

Comment:
Below this there should be a subtitle "4.2.1 DEA Signing Certificate"

Reply:
Fixed as well!